### PR TITLE
Automatic pr generator

### PR DESCRIPTION
**Pull request by algojack**
Part of pipeline to automatically generate docs repo PR whenever Indexer or go-algorand has a release.

If not specified, gets latest stable versions for go-algorand, and latest version without dashes for indexer.

Test run: https://github.com/algorand/docs/pull/873

### CODE REVIEW

**Review by barnjamin — APPROVED** [review]

**Review by algobarb — APPROVED** [review]
LGTM

File: .github/workflows/pr-generator.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+name: "PR Generator"
+on:
+  workflow_dispatch:
+    inputs:
+      go-algorand-version:
```
Comment (algolucky) [inline .github/workflows/pr-generator.yml:5]:
```suggestion
      go_algorand_version:
```

should this use this formatting instead?

**Review by algolucky — COMMENTED** [review]
I'd personally add some spacing, but that doesn't necessarily change the functionality.

**Comment by algojack** [comment]
> I'd personally add some spacing, but that doesn't necessarily change the functionality.

added.

**Review by algobarb — APPROVED** [review]

**Pull request merged**